pkgs/by-name/li/libfprint-2-tod1-broadcom/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  autoPatchelfHook,
+  fetchzip,
+  lib,
+  libfprint-tod,
+  openssl,
+  patchelfUnstable,
+  stdenv,
+}:
+
+# Based on ideas from (using a wrapper library to redirect fopen() calls to firmware files):
+#   * https://tapesoftware.net/replace-symbol/
+#   * https://github.com/NixOS/nixpkgs/pull/260715
+let
+  version = "5.12.018";
+
+  unpacked = stdenv.mkDerivation {
+    pname = "libfprint-2-tod1-broadcom-unpacked";
+    inherit version;
+
+    src = fetchzip {
```

Comment:
If this is the same package (and therefore the same `src`), wouldn't it make sense to just put the whole  `src = ` statement inside the `let` statement (just like `version`) and inherit it for `unpacked`  and `libfprint-2-tod1-broadcom`?

Reply:
Good point, thanks!